_posts/2024-05-15-Bring-something-green-to-EMF.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Bring somethig green to EMF!"
+---
+
+# Bring something green to EMF!
+Do you have a favourite house plant or herb?  Have you been planting seeds or cuttings this spring?  Would you like to share or swap, or just to take something green and new back home with you as a novice gardener?
+
+We’re excited to invite you to contribute to the Community Cultivation Collaboration (the CCC), a space to which anyone can bring small cuttings, offshoots, seedlings, bulbs or spare seeds for everyone to browse, and select from to take back home.
+
+![A plant growing in a glass beaker held up against the light; ready to go to EMF](../../../../images/2024-05-ccc-pot-plant.png "A plant for EMF")
+
+## The practical details
+
+For the CCC to work we both need you to bring plants & take them home with you
+
+### Bringing plants to Electromagnetic Field 2024
+
+* If you can bring your shares and swaps in small non-glass containers or pots, that would be ideal - but, if transport space is tight, pack your swaps in plastic bags as we will have some containers available.  
+* Please label what you have brought (surprises are lovely, but …).
+* Don’t forget to keep an eye on the hydration of any growing offerings you’ve brought.
```

Comment:
I’m not entirely clear what this is asking. Should people make they’re hydrated in transit, or do they also need to check in now and again on site?

Reply:
The latter I think